package.json

```diff
@@ -6,17 +6,30 @@
     "dev": "vite",
     "build": "tsc && vite build",
     "preview": "vite preview",
-    "prod": "vite build --mode=production && vite preview"
+    "prod": "vite build --mode=production && vite preview",
+    "test": "jest",
+    "test:watch": "jest --watch"
   },
-	"dependencies": {
-		"@nypl/design-system-react-components": "1.5.1",
-		"react": "17.0.2",
-		"react-dom": "17.0.2"
-	},
-	"devDependencies": {
-		"@types/node": "^18.13.0",
-		"@types/react": "16.9.41",
-		"@types/react-dom": "^16.9.0", 
+  "dependencies": {
+    "@nypl/design-system-react-components": "1.5.2",
+    "js-cookie": "^3.0.1",
```

Comment:
For now, can you remove all the carets? It should not be a big problem but I want to avoid any patches or bug fix updates that can cause the test or build to fail. If the packages are pinned, we know exactly the package version that is working or not working.